.multi-releaserc

```diff
@@ -0,0 +1,9 @@
+{
+    "ignorePrivate": false,
+    "ignorePackages": ["!packages/**"],
+    "deps": {
+        "bump": "ignore",
+        "release": "patch",
+        "prefix": "^"
+    }
```

Comment:
Can you explain what each of these settings do?

Reply:
https://github.com/anolilab/semantic-release/tree/main/packages/multi-semantic-release#configuring-multi-semantic-release

Reply:
`ignorePrivate: false,`: Our packages/projects like bots, and API doesn't need to be published on any registry, so we set `"private": true` in their `package.json`s, but we still need to build them.  
`ignorePackages: [...]`: We don't need to release libraries in `packages/`.  

`deps.bump: "ignore"`: We don't need to autobump when dependencies update because all projects currently target the version `*` (any version).  
`deps.release` and `deps.prefix`: These are other autobump configs which can be utilized later on down the line if we ever need autobump.